src/BlockTypes/CollectionStockFilter.php

```diff
@@ -77,16 +77,18 @@ protected function render( $attributes, $content, $block ) {
 	 */
 	private function get_stock_filter_html( $stock_counts, $attributes ) {
 		$display_style  = $attributes['displayStyle'] ?? 'list';
+		$show_counts    = $attributes['showCounts'] ?? false;
 		$stock_statuses = wc_get_product_stock_status_options();
 
 		// check the url params to select initial item on page load.
 		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce verification is not required here.
 		$selected_stock_status = isset( $_GET[ self::STOCK_STATUS_QUERY_VAR ] ) ? sanitize_text_field( wp_unslash( $_GET[ self::STOCK_STATUS_QUERY_VAR ] ) ) : '';
 
 		$list_items = array_map(
-			function( $item ) use ( $stock_statuses ) {
+			function( $item ) use ( $stock_statuses, $show_counts ) {
+				$label = $show_counts ? $stock_statuses[ $item['status'] ] . ' ( ' . $item['count'] . ' ) ' : $stock_statuses[ $item['status'] ];
```

Comment:
I think we don't need space here. 
```suggestion
				$label = $show_counts ? $stock_statuses[ $item['status'] ] . ' (' . $item['count'] . ') ' : $stock_statuses[ $item['status'] ];
```

Reply:
@dinhtungdu this was to match the CSS that adds before/after with spaces. As I mentioned in previous comment. I could remove spacing in both cases, but this was consistency with the original I think

Reply:
@samueljseay I'm see the different behavior with the current filter blocks, here is the example of attribute filter with count enabled.

In the editor, there is no padding/spacing: 
![image](https://github.com/woocommerce/woocommerce-blocks/assets/5423135/1e389c17-8171-4299-b6e6-46260d6c0e22)


On the frontend:

![image](https://github.com/woocommerce/woocommerce-blocks/assets/5423135/907583c5-8558-49db-a74e-7d17fb266265)

Reply:
@dinhtungdu ok cool, I'll just change it to match. i did think it was weird :D

Reply:
Resolved [here](https://github.com/woocommerce/woocommerce-blocks/pull/11827/commits/f78484b8093cf5db3b28ef8f26526a47455a41c5)